libs/island-ui/core/src/lib/Toast/Toast.tsx

```diff
@@ -74,7 +74,7 @@ export const ToastContainer: React.FC<ToastProps> = ({
         pauseOnHover
         transition={Slide}
       />
-      {useKeyframeStyles && <style jsx>{toastKeyframes}</style>}
+      {useKeyframeStyles && <style>{toastKeyframes}</style>}
```

Comment:
Why are you removing this? There is code in this file (line 13) that explicitly adds the ability for this to be set on the style tag.

Reply:
This was causing warnings in the console, I did not notice any breaking changes after removing this. If this has a special meaning for a specific framework maybe we should look into moving this to treat? or we could try to find a better way to remove the warning 😄 

![image](https://user-images.githubusercontent.com/5655741/111303712-bce9af80-864c-11eb-91a7-b93327752adc.png)

Reply:
I'll revert this for now and move the transitions to treat in a separate PR

Reply:
@oddsson https://github.com/island-is/island.is/pull/3264